### ISSUE TITLE
Animate plots

### ISSUE THREAD

**Issue by codeOfRobin** [aleph7/PlotKit#12]
Like change data points and stuff?

**Comment by alejandro-isaza** [comment]
Not right now, but that would be a nice feature to have.

**Comment by codeOfRobin** [comment]
I'm gonna give it a shot, but one thing that really bugs me is that Core Animation in AppKit isn't as fully featured as UIKit. Is there any drop in replacement for 

```
UIView.animate(duration:,delay.....{
// Add final states here in the closure
})
```

?

Meanwhile, any good resources on Core Animation in OSX?

**Comment by alejandro-isaza** [comment]
I don't think that would be right approach anyway. PlotView has it's own drawing code and it's own way of specifying plots. I think it would have to be something like

``` swift
plotView.animate(fromPointSet: a, toPointSet: b)
```

and then have something that interpolates between points in `a` and points in `b` 60 times a second.
